itests/sector_terminate_test.go

```diff
@@ -18,10 +17,6 @@ import (
 )
 
 func TestTerminate(t *testing.T) {
-	if os.Getenv("LOTUS_TEST_WINDOW_POST") != "1" {
-		t.Skip("this takes a few minutes, set LOTUS_TEST_WINDOW_POST=1 to run")
-	}
-
```

Comment:
Those guards are still really useful when running tests locally. If we don't want per-test env vars, I'd either unify them (only have a single env var like `LOTUS_RUN_EXPENSIVE_TESTS`), or just check for the `CI` env var

Reply:
Done. Added a `kit.Expensive(t)` function that marks a test as expensive and skips it if running outside CI and the env var is not set.